raysort/sort_utils.py

```diff
@@ -290,6 +300,23 @@ def validate_part(cfg: AppConfig, pinfo: PartInfo) -> Tuple[int, bytes]:
     return ret
 
 
+def compare_checksums(input_checksums: List[str], output_summary: str) -> bool:
+    input_checksum = sum(input_checksums)
+    input_checksum = str(hex(input_checksum))[-16:]
+    output_attrs = output_summary.split("\\n")
+
+    assert len(output_attrs) == 5, output_summary
```

Comment:
probably easier to define `checksum_str = output_attrs[1]` so that we don't have to keep referring to `output_attrs[1]`.

Reply:
Removed completely